### PR TITLE
Create ProblemsMotivations.md

### PR DESCRIPTION
**Pull request by molnard**

### CODE REVIEW

File: ProblemsMotivations.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+This is a list that collets problems and motivations related crypto wallets in general. This is not focusing on a specific implementation.
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
This is a list that collects problems and motivations related to crypto wallets in general. This is not focusing on a specific implementation.
```

File: ProblemsMotivations.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+While we already have a working wallet it is worth to get back to the basics and re-evaluate: **What are the problems we want to solve in real life with a crypto wallet?**
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
While we already have a working wallet it is important to get back to the basics and re-evaluate: **What are the problems we want to solve in real life with a crypto wallet?**
```

File: ProblemsMotivations.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+So now please sit down and write 5 short sentences that start with, "My problem is, ..."
+
+Helper questions: 
+
+Regarding crypto...
+
+- What is your the biggest problem?
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- What is your biggest problem?
```

File: ProblemsMotivations.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+- Tell me your 3 most annoying problems?
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- What is your top three most annoying problems?
```

File: ProblemsMotivations.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+- How and where can I buy bitcoin?
+
+- Where can I get some help if I have problems?
+- Who is storing my money?
+
+- I don't know who to trust?
+- Which software should I use?
+
+- Why bitcoin?
+- How it is different from traditional banking system?
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- How is it different from the traditional banking system?
```

File: ProblemsMotivations.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+- Should I buy bitcoin or altcoins?
+
+- I cannot use the softwares, don't know what to do there.
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- I cannot use the software, I don't know what to do there.
```

File: ProblemsMotivations.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+- I don't understand, I don't speak english.
+- How much fiat do I have currenly in my wallet?
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- How much fiat do I have currently in my wallet?
```

File: ProblemsMotivations.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+- I am not sure that I am doing things in a proper way. My workflow is good or not?
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- I am not sure that I am doing things in a proper way. Is my workflow good or not?
```

File: ProblemsMotivations.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+
+- What happens if I lost my device or stolen?
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- What happens if I lose my device or it gets stolen?
```

File: ProblemsMotivations.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+- I don't know if my wallet is safe if somebody gets my laptop.
+- I am not sure that I have writted down correcly my mnemonics and stored it in a safe way.
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- I am not sure that I have written down correctly my mnemonics and stored it in a safe way.
```

File: ProblemsMotivations.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+- I am not sure that I have backed up my wallet properly. 
+
+- I don't know what kind of private infomations am I sharing with who, when I am using a wallet. (comparison table)
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- I don't know what kind of private information am I sharing with who, when I am using a wallet. (comparison table)
```

File: ProblemsMotivations.md [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+
+- I have to verify a lot of numbers when copy/paste bitcoin addresses, scary.
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- I have to verify a lot of characters when I copy/paste bitcoin addresses, which is scary.
```

File: ProblemsMotivations.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+- I didn't know that I have coins.
+
+- I don't know the exact fees. 
+- Confirmation time and fees how is it related. 
+
+- Badly designed wallets. They are ugly. 
+- The UI of the wallets are not good.
+
+- Different units used those are unknows sats, bitcoin.
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- Different units used that are unknown sats, bitcoin.
```

File: ProblemsMotivations.md [hunk 14]
```diff
@@ -0,0 +1,76 @@
...
+- Unknown and not standardized terms used in different softwares.
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- Unknown and not standardized terms used in different software.
```

File: ProblemsMotivations.md [hunk 15]
```diff
@@ -0,0 +1,76 @@
...
+
+- I don't know the exact fees. 
+- Confirmation time and fees how is it related. 
+
+- Badly designed wallets. They are ugly. 
+- The UI of the wallets are not good.
+
+- Different units used those are unknows sats, bitcoin.
+- Unknown and not standardized terms used in different softwares. 
+- I have to send my address every time I receive, complicated.
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- I have to send my address every time I receive, complicated.
```

File: ProblemsMotivations.md [hunk 16]
```diff
@@ -0,0 +1,76 @@
...
+
+- Badly designed wallets. They are ugly. 
+- The UI of the wallets are not good.
+
+- Different units used those are unknows sats, bitcoin.
+- Unknown and not standardized terms used in different softwares. 
+- I have to send my address every time I receive, complicated. 
+
+- I want to have only one wallet with one backup, but somehow different profiles to separate businesses.
+- I want to have only one wallet even among devices (mobile phone and desktop)
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- I want to have only one wallet even among devices (mobile phone and desktop).
```

File: ProblemsMotivations.md [hunk 17]
```diff
@@ -0,0 +1,76 @@
...
+- I have to test the wallet in live with real money. What if I do something wrong and loose that money?
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- I have to test the wallet in live with real money. What if I do something wrong and lose that money?
```

File: ProblemsMotivations.md [hunk 18]
```diff
@@ -0,0 +1,76 @@
...
+
+- I had to study bitcoin before the usage, learning curve.
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- I had to study Bitcoin before using it, learning curve.
```

File: ProblemsMotivations.md [hunk 19]
```diff
@@ -0,0 +1,76 @@
...
+
+- I want to have only one wallet with one backup, but somehow different profiles to separate businesses.
+- I want to have only one wallet even among devices (mobile phone and desktop)
+- I have to test the wallet in live with real money. What if I do something wrong and loose that money?
+
+- I had to study bitcoin before the usage, learning curve. 
+
+- I am not able to personalize the feature in my wallet.
+- I cannot use the same wallet software on every platform (desktop and mobile).
+- Interoperatibility between different wallets, bisq?
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- Interoperability between different wallets, bisq?
```

File: ProblemsMotivations.md [hunk 20]
```diff
@@ -0,0 +1,76 @@
...
+
+- I cannot use the softwares, don't know what to do there.
+- I don't understand, I don't speak english.
+- How much fiat do I have currenly in my wallet?
+- I am not sure that I am doing things in a proper way. My workflow is good or not?
+
+- What happens if I lost my device or stolen?
+- I don't know if my wallet is safe if somebody gets my laptop.
+- I am not sure that I have writted down correcly my mnemonics and stored it in a safe way.
+- I am not sure that I have backed up my wallet properly.
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- I am not sure that I have backed up my wallet properly.
```

File: ProblemsMotivations.md [hunk 21]
```diff
@@ -0,0 +1,76 @@
...
+
+- I don't know what kind of private infomations am I sharing with who, when I am using a wallet. (comparison table)
+
+- I have to verify a lot of numbers when copy/paste bitcoin addresses, scary.
+- I didn't know that I have coins.
+
+- I don't know the exact fees. 
+- Confirmation time and fees how is it related. 
+
+- Badly designed wallets. They are ugly.
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- I don't know the exact fees.
- Confirmation time and fees how is it related.

- Badly designed wallets. They are ugly.
```

File: ProblemsMotivations.md [hunk 22]
```diff
@@ -0,0 +1,76 @@
...
+
+- I am not able to personalize the feature in my wallet.
+- I cannot use the same wallet software on every platform (desktop and mobile).
+- Interoperatibility between different wallets, bisq?
+
+# Motivations
+
+- I want to invest in bitcoin.
+- I want to store my money. 
+- I want my financial privacy back.
```
Comment (yahiheb) [inline ProblemsMotivations.md]:
```suggestion
- I want to store my money.
- I want my financial privacy back.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: ProblemsMotivations.md (hunk 7 above)
Comment (ghost) [reply ProblemsMotivations.md]:
```suggestion
- How much are the bitcoin in my wallet currently worth in fiat?
```

File: ProblemsMotivations.md [hunk 23]
```diff
@@ -6,9 +6,33 @@ So now please sit down and write 5 short sentences that start with, "My problem
...
+
+- I don't know who to trust?
+- Which software should I use?
+
+- Why bitcoin?
+- How it is different from traditional banking system?
+- Should I buy bitcoin or altcoins?
+
+- I cannot use the softwares, don't know what to do there.
+- I don't understand, I don't speak english.
```
Comment (ghost) [inline ProblemsMotivations.md]:
```suggestion
- I don't understand specific terminology.
- I don't speak english.
```

File: ProblemsMotivations.md [hunk 24]
```diff
@@ -6,9 +6,33 @@ So now please sit down and write 5 short sentences that start with, "My problem
...
+- How much fiat do I have currenly in my wallet?
+
+- What happens if I lost my device or stolen?
+
```
Comment (ghost) [inline ProblemsMotivations.md:65]:
```suggestion

- How can I segregate my transaction history?
- What are statistics on my receiving and spending habits? 
```

**Comment by molnard** [comment]
Thx @yahiheb and @MaxHillebrand

**Pull request merged**